src/Servers/IIS/src/AspNetCoreModuleV2/CommonLib/exceptions.h

```diff
@@ -74,6 +74,7 @@
 #define LOG_IF_FAILED(hr)                                       LogHResultFailed(LOCATION_INFO, hr)
 #define LOG_LAST_ERROR()                                        LogLastErrorIf(LOCATION_INFO, true)
 #define LOG_LAST_ERROR_IF(condition)                            LogLastErrorIf(LOCATION_INFO, condition)
+#define LOG_LAST_ERROR_IF_NOT(condition)                        !LogLastErrorIf(LOCATION_INFO, !condition)
```

Comment:
This seems wrong, I think you only need to invert once.

Reply:
Idea is to get original condition value but log error if it's not true, maybe I should remove it.

Reply:
Yeah I'd say remove it, too confusing.